Resources/Prototypes/Entities/Objects/Tools/toolbox.yml

```diff
@@ -11,12 +11,12 @@
   - type: Storage
     capacity: 60
   - type: Item
-    size: 9999
+    size: 110
```

Comment:
Tests are failing cause of this. Syndie toolbox can contain items up to 170 weight, while its own weight is 110

Reply:
fixed